include/seqan3/core/configuration/configuration.hpp

```diff
@@ -571,6 +351,37 @@ template <detail::config_element config_t>
 configuration(config_t) -> configuration<config_t>;
 //!\}
 
+/*!\brief Combines configuration and configuration elements to create a new configuration object.
```

Comment:
Just a small detail. Are "configuration" and "configuration object" used interchangeably to mean a seqan3::configuration object? Found a terminology here [/seqan/product_backlog/issues/131](https://github.com/seqan/product_backlog/issues/131) that doesn't mention this

Reply:
I see your point here. So yes, it does but it could be clearer here I guess. I am trying to update the PR today. Thank you.

Reply:
Ok, I changed only a little bit the documentation. Do you think it is clear now?

Reply:
Where is this change visible?

Reply:
Oh, there was an error during the push so it wasn't published yet. Now the rebased version should contain the change.

Reply:
Ah yes, that clears it. Thanks!